samples/scenarios/vendor_attributes/postgres_vendor_attributes/manifest.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version='1.0' encoding='utf-8' ?>
```

Comment:
Should this postgres_vendor_attributes folder be included?  Looks the same as postgres_vendor.

Reply:
I renamed the folder, for some reason not all the files were removed. Fixed in latest commit.